src/servers/src/mysql/helper.rs

```diff
@@ -0,0 +1,238 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+use std::ops::ControlFlow;
+use std::time::Duration;
+
+use chrono::{NaiveDate, NaiveDateTime};
+use common_query::prelude::ScalarValue;
+use datatypes::prelude::ConcreteDataType;
+use datatypes::value::{self, Value};
+use itertools::Itertools;
+use opensrv_mysql::{ParamValue, ValueInner};
+use snafu::ResultExt;
+use sql::ast::{visit_expressions_mut, Expr, Value as ValueExpr, VisitMut};
+use sql::statements::statement::Statement;
+
+use crate::error::{self, Result};
+
+/// Returns the placeholder string "$i".
+pub fn format_placeholder(i: usize) -> String {
+    format!("${}", i)
+}
+
+/// Replace all the "?" placeholder into "$i" in SQL,
+/// returns the new SQL and the last placeholder index.
+pub fn replace_placeholders(query: &str) -> (String, usize) {
```

Comment:
Can this method be replaced by `transform_placeholders`?

Reply:
No, it's used in other queries such as create table etc.